plugins/ReverbSC/CMakeLists.txt

```diff
@@ -7,18 +7,21 @@ SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
 
 BUILD_PLUGIN(
 	reverbsc
-	ReverbSC.cpp 
-	ReverbSCControls.cpp 
-	ReverbSCControlDialog.cpp 
+	ReverbSC.cpp
+	ReverbSCControls.cpp
+	ReverbSCControlDialog.cpp
 	base.c
 	revsc.c
+	base.h
+	revsc.h
+	dcblock.h
 	dcblock.c
-	MOCFILES 
 	ReverbSC.h
-	ReverbSCControls.h 
-	ReverbSCControlDialog.h 
+	MOCFILES
+	ReverbSCControls.h
+	ReverbSCControlDialog.h
 	base.h
 	revsc.h
 	dcblock.h
-	EMBEDDED_RESOURCES artwork.png logo.png
+	EMBEDDED_RESOURCES "${CMAKE_CURRENT_SOURCE_DIR}/*.png"
```

Comment:
And this line shouldn't be changed, because of https://github.com/LMMS/lmms/commit/9f905bce3eee4e24a2338d9d8a104af6bdbec8ad#diff-ea4a8d6b4968ac81d0ffb07a45a4c7e6 in #1891.